nservicebus/azure-storage-queues/configuration.md

```diff
@@ -95,27 +91,7 @@ WARNING: Changing the value of `DegreeOfReceiveParallelism` will influence the t
 
 WARNING: The values of `BatchSize` , `DegreeOfParallelism`, `Concurrency`, [ServicePointManager Settings](/nservicebus/azure-storage-persistence/performance-tuning.md) and the other parameters like `MaximumWaitTimeWhenIdle` have to be selected carefully in order to get the desired speed out of the transport while not exceeding [the boundaries](https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/#storage-limits) of the allowed number of operations per second.
 
-NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify the endpoint name and select a scale-out option.
-
-Parameters' values can be configured in the following ways:
-
-
-### Via the configuration API
-
-In Versions 7 and below the default settings can be overridden only using configuration API:
-
-snippet:AzureStorageQueueConfigCodeOnly
-
-
-### Via the App.Config
-
-In Versions 5 and 6 all settings can be overridden by adding to the `web.config` or the `app.config` files a configuration section called `AzureQueueConfig`:
-
-snippet:AzureStorageQueueConfig
-
-Note that the connection string can be also configured by specifying a value for connection string called `NServiceBus/Transport`, however this value will be overridden if another is provided in `AzureServiceBusQueueConfig`:
-
-snippet: AzureStorageQueueConnectionStringFromAppConfig
+partial:config
```

Comment:
The note about `QueueName` and `QueuePerInstance` was obsoleted. This was relevant in v6 of the transport. Should probably stain with v6 partial.